redpipe/connections.py

```diff
@@ -135,6 +154,7 @@ def disconnect(cls, name: Optional[str] = None) -> None:
         """
         try:
             del cls.connections[name]
+            del cls.clients[name]
```

Comment:
FWIW, it also works here as expected (but still weird)
<img width="163" alt="image" src="https://github.com/user-attachments/assets/ad663779-4716-4673-8cf5-af227032d7c6">